home/models.py

```diff
@@ -22,6 +22,15 @@ class Group(models.Model):
     def __str__(self):
         return self.name
 
+    def number_users(self):
+        return self.profiles.count()
+    
+    def number_solved_problems(self):
+        return reduce(
```

Comment:
Would be nice the code have a standard. Because we have functions like " number_of_x", "get_amount_of_x". What do you think?

Reply:
I agree, what standard do you prefer ? I suggest to use  get_amount_of_x because you used this pattern already.

Reply:
Don't care about change the project if you have better solutions.
I think we should follow the conventions whenever we can.
In this order: **language > framework > project**.

This brings familiarity for new people that are joinning to the project.

Maybe if we use `count_x` like the django *querySet* api may be a good idea.

Do you know some api that has the same purpose?

Reply:
I will change, it's good for me too.
When we using `count_x` have a similarity also with database queries too.
I don't know any api who use something different.